vespa-hadoop/src/main/java/com/yahoo/vespa/hadoop/mapreduce/VespaRecordWriter.java

```diff
@@ -143,4 +154,28 @@ private static int streamsPerConnection(VespaConfiguration config) {
                 .map(hostname -> URI.create(String.format("https://%s:%d/", hostname, config.defaultPort())))
                 .collect(toList());
     }
+
+    private static class DryrunClient implements FeedClient {
```

Comment:
Let's move this to the json parser package. I think we should offer some dry-run option for just verifying the JSON, counting documents, etc., there.